### DISCUSSION TITLE
2fa authentication isn't available in my country<title>

### DISCUSSION THREAD

**Discussion by MahmoudNobani** [community/community#78776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hey, i am from Palestine and i am trying to Enable two-factor authentication (2FA), the problem is that i cant get the token because the region code is not available for Palestine (+970), what should i do in this case, who can i connect with to solve the issue

**Comment by ghostinhershell** [comment]
Hey there @MahmoudNobani, 

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
